### PR TITLE
Add Nextflow icon to badgen

### PR DESCRIPTION
**Pull request by mribeirodantas**
[Nextflow](https://github.com/nextflow-io/nextflow) enables scalable and reproducible scientific workflows using software containers. It allows the adaptation of pipelines written in the most common scripting languages. It's a tool used daily by dozens of thousands of users worldwide with a large number of GitHub repositories that would benefit from such a badge 🥳

### CODE REVIEW

**Comment by mribeirodantas** [comment]
@walpox can you merge this, please? 🙂

**Comment by walpox** [comment]
> @walpox can you merge this, please? 🙂

Hello,
I am afraid I cannot help you as I am a regular contributor without the required permissions to merge your changes.

**Comment by mribeirodantas** [comment]
Do you know who I should mention to get it merged? 😬

**Comment by mribeirodantas** [comment]
@amio ? 😅

**Review by styfle — CHANGES_REQUESTED** [review]
The green fill needs to be removed. See the other icons for an example.

https://badgen-icons-git-fork-mribeirodantas-add-nxf-icon-badgen.vercel.app/

image

**Comment by mribeirodantas** [comment]
Thanks for replying, @styfle. It's done now.

File: icons/nextflow.svg [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<svg
+   width="251"
+   height="251"
+   viewBox="0 0 251 251"
+   fill="none"
+   version="1.1"
+   id="svg317"
+   sodipodi:docname="nextflow.svg"
+   inkscape:version="1.2 (dc2aeda, 2022-05-15)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
```
Comment (styfle) [inline icons/nextflow.svg:11]: Lets remove all the inkscape specific code

**Comment by mribeirodantas** [comment]
Done, @styfle. Thanks again 😉

File: icons/nextflow.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="251" height="251" fill="none"><path fill="#0DC09D" d="M0 47.02c39.45 2.62 71.06 34.307 73.54 73.795h46.07C117.05 55.859 64.93 3.642 0 .943V47.02ZM73.8 131.324c-2.62 39.447-34.31 71.055-73.8 73.535v46.067c64.96-2.56 117.18-54.677 119.88-119.602H73.8ZM176.201 120.545c2.62-39.448 34.31-71.056 73.8-73.535V.943c-64.96 2.56-117.18 54.676-119.88 119.602h46.08ZM250.001 204.849c-39.45-2.62-71.06-34.307-73.54-73.795h-46.07c2.56 64.956 54.68 117.172 119.61 119.872v-46.077Z" style="fill:#fff;fill-opacity:1"/></svg>
```
Comment (styfle) [inline icons/nextflow.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="251" height="251"><path fill="#0DC09D" d="M0 47.02c39.45 2.62 71.06 34.307 73.54 73.795h46.07C117.05 55.859 64.93 3.642 0 .943V47.02ZM73.8 131.324c-2.62 39.447-34.31 71.055-73.8 73.535v46.067c64.96-2.56 117.18-54.677 119.88-119.602H73.8ZM176.201 120.545c2.62-39.448 34.31-71.056 73.8-73.535V.943c-64.96 2.56-117.18 54.676-119.88 119.602h46.08ZM250.001 204.849c-39.45-2.62-71.06-34.307-73.54-73.795h-46.07c2.56 64.956 54.68 117.172 119.61 119.872v-46.077Z"/></svg>
```

File: icons/nextflow.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="251" height="251"><path fill="#0DC09D" d="M0 47.02c39.45 2.62 71.06 34.307 73.54 73.795h46.07C117.05 55.859 64.93 3.642 0 .943V47.02ZM73.8 131.324c-2.62 39.447-34.31 71.055-73.8 73.535v46.067c64.96-2.56 117.18-54.677 119.88-119.602H73.8ZM176.201 120.545c2.62-39.448 34.31-71.056 73.8-73.535V.943c-64.96 2.56-117.18 54.676-119.88 119.602h46.08ZM250.001 204.849c-39.45-2.62-71.06-34.307-73.54-73.795h-46.07c2.56 64.956 54.68 117.172 119.61 119.872v-46.077Z"/></svg>
```
Comment (styfle) [inline icons/nextflow.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="251" height="251"><path fill="#fff" d="M0 47.02c39.45 2.62 71.06 34.307 73.54 73.795h46.07C117.05 55.859 64.93 3.642 0 .943V47.02ZM73.8 131.324c-2.62 39.447-34.31 71.055-73.8 73.535v46.067c64.96-2.56 117.18-54.677 119.88-119.602H73.8ZM176.201 120.545c2.62-39.448 34.31-71.056 73.8-73.535V.943c-64.96 2.56-117.18 54.676-119.88 119.602h46.08ZM250.001 204.849c-39.45-2.62-71.06-34.307-73.54-73.795h-46.07c2.56 64.956 54.68 117.172 119.61 119.872v-46.077Z"/></svg>
```

**Review by styfle — APPROVED** [review]
I pushed a few more changes, thanks!

**Comment by mribeirodantas** [comment]
Does it take a while for the website to be able to render? The icon is not appearing here yet: https://badgen.net/badge/Nextflow/23.10.1/green?icon=nextflow

**Comment by styfle** [comment]
It deployed here: https://icons.badgen.net

The root badgen.net is a separate repo here: https://github.com/badgen/badgen

**Comment by mribeirodantas** [comment]
Can you help me with instructions on how I can make the nextflow icon be found in badgen.net, just like the GitHub one?

**Comment by amio** [comment]
Thanks @mribeirodantas @styfle 

> Can you help me with instructions on how I can make the nextflow icon be found in badgen.net, just like the GitHub one?

Currently we need to release a new version of this npm package, then update the dependency in badgen.net.
I think we better migrate this repo into https://github.com/badgen/badgen.net, to prevent more confusions in future.

**Pull request merged**